application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -54,20 +62,18 @@ private boolean messageHasNoMediaAttached(Message message) {
                 && !message.getContentRaw().contains("http");
     }
 
-    private RestAction<Message> dmUser(Message message) {
+    private MessageCreateData warningMessage(Message message) {
         String originalMessageContent = message.getContentRaw();
+
         MessageEmbed originalMessageEmbed =
                 new EmbedBuilder().setDescription(originalMessageContent)
                     .setColor(Color.ORANGE)
                     .build();
 
-        MessageCreateData dmMessage = new MessageCreateBuilder().setContent(
-                "Hey there, you posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
+        return new MessageCreateBuilder().setContent("Hey there, you "
+                + message.getAuthor().getAsMention()
+                + " posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
```

Comment:
reads weird with the mention right in the text. put it at the very front instead. `"@Foo Hey there, you posted a ..."`

Reply:
Ok I will update the Text

Reply:
u forgot the space

Reply:
done I updated it